build.gradle

```diff
@@ -3,16 +3,16 @@ plugins {
     id 'checkstyle'
     id 'pmd'
     id 'jacoco'
-    id 'com.github.johnrengelman.shadow' version '6.1.0'
-    id 'nebula.lint' version '17.7.0'
```

Comment:
why did you remove nebula lint? It might be good to lint our gradle files? what do you think?